launch/driver.launch

```diff
@@ -65,6 +65,14 @@
     xyzir
     }"/>
 
+  <arg name="azimuth_window_start" default="0" doc="azimuth window start;
+    values range [0, 360000] millideggrees"/>
+  <arg name="azimuth_window_end" default="360000" doc="azimuth window end;
+    values range [0, 360000] millideggrees"/>
```

Comment:
Using a default of `0` and `360000` can be breaking change for some users since it will reset their current Azimuth Window setting. We can avoid this by using `-1` or any negative value as to no change, or use a empty string value similar what other options we had.

Reply:
Nit, spelling: `millidegrees`